README.md

```diff
@@ -27,6 +27,23 @@ centrally managing routing policy and fronting service discovery.
 * [linkerd](linkerd/README.md)
 * [namerd](namerd/README.md)
 
+## Quick start ##
+
+### Boot linkerd ###
+
+```
+$ ./sbt linkerd-examples/http:run
+$ open http://localhost:9990
+```
+
+### Boot namerd ###
+
+```
+$ ./sbt namerd-examples/basic:run
+$ open http://localhost:9991
+$ curl :4180/api/1/dtabs
+```
```

Comment:
selfishly i find this quite helpful. i think for the uninitiated, it gets folks up and running very quickly, without having to know much about sbt. i'd like developers to go from clone to admin page in as few seconds as possible.